scipy/interpolate/_bsplines.py

```diff
@@ -1402,7 +1402,8 @@ def make_interp_spline(x, y, k=3, t=None, bc_type=None, axis=0,
 
 
 def make_lsq_spline(x, y, t, k=3, w=None, axis=0, check_finite=True):
-    r"""Compute the (coefficients of) an LSQ B-spline.
+    r"""Compute the (coefficients of) an LSQ (Least SQuared) based
+    fitting B-spline.
```

Comment:
I think we should state clearly that this API for fitting.

Reply:
Nit, optional: While we're at it, can delete the braketed (coefficients of) part. This function actually returns a b-spline, not it's coefficients. This line dates back to earlier iterations when there was an idea to return tck tuples